sematic/cli/settings.py

```diff
@@ -1,45 +1,70 @@
+# Standard Library
+import sys
+
 # Third-party
 import click
-import yaml
 
 # Sematic
 from sematic.cli.cli import cli
-from sematic.user_settings import SettingsVar, get_all_user_settings, set_user_settings
+from sematic.config.user_settings import (
+    UserSettingsVar,
+    delete_user_settings,
+    dump_settings,
+    get_active_user_settings,
+    set_user_settings,
+)
 
 
 @cli.group("settings")
-def settings():
+def settings() -> None:
     pass
 
 
-@settings.command("show", short_help="Show active settings")
-def show_settings():
+@settings.command("show", short_help="Show the currently active settings")
+def show_settings() -> None:
     """
-    Show currently active settings.
+    Show the currently active settings.
     """
-    settings = get_all_user_settings()
-    click.echo("Active settings:\n")
-    click.echo(yaml.dump(settings, default_flow_style=False))
+    profile_settings_dump = dump_settings(get_active_user_settings())
+    click.echo(f"Active settings:\n{profile_settings_dump}")
 
 
-@settings.command("set", short_help="Set settings value")
+@settings.command("set", short_help="Set a setting value")
 @click.argument("var", type=click.STRING)
 @click.argument("value", type=click.STRING)
-def set_settings_cli(var, value):
+def set_settings(var: str, value: str) -> None:
     """
-    Set a settings value
+    Set a setting value.
```

Comment:
I feel meh about using `setting` as a singular noun. I though it was always plural, but I couldn't find any good reference to back that claim so 🤷

Reply:
Yeah, I'm torn as well. "Setting" seems to be the correct term, but it's rarely used in common speaking. Reverting to "settings".